06_timers/userspace_script/show_time.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <time.h>
+
+int main(void)
+{
+	struct timespec now = {0};
+	struct tm *hrt = {0};		// human readable time
+
+	clock_gettime(0, &now);
+	hrt = localtime(&now.tv_sec);
+
+	printf("It is %d:%d:%d %d-%d-%d\n",
+			hrt->tm_hour, hrt->tm_min, hrt->tm_sec,
+			hrt->tm_year + 1900,
+			hrt->tm_mon, hrt->tm_mday);
```

Comment:
According to struct tm specification:
int tm_mon;         /* month, range 0 to 11             */
So in this case printf shows wrong month